fibonacci.asm

```diff
@@ -18,14 +18,23 @@
 
 fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
 size:	.word  9              # create a single integer variable named "size" that indicates the length of the array
+input: .asciiz "\Enter size of the array\n"
 
 ###############################################
 # .text segment. Assembly instructions go here.
 ###############################################
 	.text
+              la   $a0, input
+              li   $v0, 4
+              syscall
+              
+              li   $v0, 5
+              syscall
+              
+              move $s5, $v0
```

Comment:
-5 Accepts input greater than 9.  This allows storing numbers beyond the bounds of the array.